JS/Lesson22/onder/script.js

```diff
@@ -0,0 +1,8 @@
+let acc = document.getElementsByClassName("accordion");
```

Comment:
Since the value of this variable is never change it's better to declare it using `const` keyword:

```suggestion
const acc = document.getElementsByClassName("accordion");
```